Sources/StreamChat/Models/Message.swift

```diff
@@ -217,6 +219,16 @@ extension _ChatMessage {
     public var isPinned: Bool {
         pinDetails != nil
     }
+    
+    public var lastActiveThreadParticipant: _ChatUser<ExtraData.User>? {
+        func sortingCriteriaDate(_ user: _ChatUser<ExtraData.User>) -> Date {
+            user.lastActiveAt ?? user.userUpdatedAt
+        }
+        
+        return threadParticipants
+            .sorted(by: { sortingCriteriaDate($0) > sortingCriteriaDate($1) })
+            .first
+    }
```

Comment:
I'm not sure this is needed tbh. It's very specific for the cases where you want to show just one avatar there. What if your UI requires two avatars, there? :)

Reply:
It does make sense to move it to StreamChatUI, I'll check if the same can be done for isPinned